README.md

```diff
@@ -4,8 +4,24 @@ Cincinnati is an update protocol designed to facilitate automatic updates. It de
 
 ## Quick Start
 
+Prepare custom environment variables
+
+```console
+# Please change these accordingly
+CINCINNATI_REGISTRY="https://quay.io"
+CINCINNATI_REPO="steveej/cincinnati-test-public"
```

Comment:
I don't want to point people at a personal repo.

Reply:
I was able to set up the test repositories under the redhat organization :raised_hands: